pages/selfCheckQuestions.tsx

```diff
@@ -0,0 +1,284 @@
+    Flex,
+    SimpleGrid,
+    Button,
+} from "@chakra-ui/react";
+import { Header, SelfCheckQuestionCard, Footer } from "@components";
+
+//Default question list, can be shortened as needed
+const questionList = [
+    {
+        _id: "60e642d7e4a1ae34207a92a3",
+        type: "multiple_choice",
+        question: "autem sunt eiusdolores nesciunt impedit?",
+        options: ["1", "2", "3"],
+        alphanumericInput: true,
+        questionNumber: 1,
+    },
+    {
+        _id: "60e642d7e4a1ae34207a92a4",
+        type: "multi_select",
+        question: "autem sunt eiusdolores nesciunt impedit?",
+        options: ["1", "2", "3"],
+        alphanumericInput: true,
+        questionNumber: 2,
+    },
+    {
+        _id: "60e642d7e4a1ae34207a92a5",
+        type: "short_answer",
+        question: "autem sunt eiusdolores nesciunt impedit?",
+        options: ["1", "2", "3"],
+        alphanumericInput: true,
+        questionNumber: 3,
+    },
+    {
+        _id: "60e642d7e4a1ae34207a92a6",
+        type: "long_answer",
+        question: "autem sunt eiusdolores nesciunt impedit?",
+        options: ["1", "2", "3"],
+        alphanumericInput: true,
+        questionNumber: 4,
+    },
+    {
+        _id: "60e642d7e4a1ae34207a92a7",
+        type: "slider",
+        question: "autem sunt eiusdolores nesciunt impedit?",
+        options: ["1", "2", "3", "4", "5"],
+        alphanumericInput: true,
+        questionNumber: 5,
+    },
+];
+
+//Self Check Questions React functional component
+const Home: React.FC = () => {
+    const [list, setList] = React.useState(questionList);
+    const [count, setCount] = React.useState(questionList.length + 1);
+    const newSliderRange0 = [];
+    for (let i = 1; i <= 3; i++) {
+        newSliderRange0.push(i);
+    }
+    const [sliderRange, setSliderRange] = React.useState(newSliderRange0);
+    function changeQuestionType(id, target) {
+        const newList = list.slice(0);
+        newList[newList.findIndex((e) => e._id === id)].type = target;
+
+        setList(newList);
+    }
+    function changeAlphanumeric(id, target) {
+        const newList = list.slice(0);
+        newList[newList.findIndex((e) => e._id === id)].alphanumericInput =
+            target;
+        setList(newList);
+    }
+    function addOneOption(id) {
+        const newList = list.slice(0);
+        newList[newList.findIndex((e) => e._id === id)].options = [
+            ...newList[newList.findIndex((e) => e._id === id)].options,
+            `${
+                newList[newList.findIndex((e) => e._id === id)].options.length +
+                1
+            }`,
+        ];
+        setList(newList);
+    }
+
+    function removeOneOption(id, index) {
+        const newList = list.slice(0);
+        newList[newList.findIndex((e) => e._id === id)].options.splice(
+            index,
+            1,
+        );
+        setList(newList);
+    }
+    function changeSliderOption(id, lowerBound, upperBound) {
```

Comment:
```suggestion
    function changeSliderRange(id, lowerBound, upperBound) {
```
To make this function name more clear